### ISSUE TITLE
"ניספח" -> "נספח"

### ISSUE THREAD

**Issue by eyalroz** [YoavHaifa/WorkersRightsComputer#17]
There's no 'י' (Yod) character in the word 'נספח'. It's redundant to add it even when printing without diacritics (Niqqud).

**Comment by YoavHaifa** [comment]
Solved by change in the HTML template

**Comment by eyalroz** [comment]
@YoavHaifa : The master branch[still has this](https://github.com/YoavHaifa/WorkersRightsComputer/blob/master/WorkersRights_v/HTML/letterTemplate.html). Can you be more specific?

**Comment by eyalroz** [comment]
Of course, I don't really understand what's going on with the [files in that folder](https://github.com/YoavHaifa/WorkersRightsComputer/tree/master/WorkersRights_v/HTML).

**Comment by YoavHaifa** [comment]
Hi Eyal,
I plan to do about a dozen changes in the SW in a development branch -
Before merging it to the main branch.
I will not close the issues before they are there.
Best Regards,
Yoav

**Comment by eyalroz** [comment]
@YoavHaifa : Yes, I understand, it's just that you had used the past tense so I thought you were referring to something that's already happened.
